kolibri/plugins/perseus_viewer/assets/src/views/PerseusRendererIndex.vue

```diff
@@ -18,9 +18,10 @@
           <!-- Layout notes
             - Layout12 span8 -> ~66% width on windowIsLarge
             - No other layout definitions means span will be 100%
-            - When not interactive, this should be full width
+            - If we're not allowingHints, then it should be 100% because
+              they're the reason we'd go smaller at all
           -->
-          <KGridItem :layout12="{ span: interactive ? 6 : 12 }">
+          <KGridItem :layout12="{ span: allowHints || interactive ? 6 : 12 }">
```

Comment:
`allowHints` defaults to `true` so this means that `interactive` will still not have any effect. Needs to be an `&&`.

https://github.com/learningequality/kolibri-design-system/blob/main/lib/content/mixin.js#L103